.buildkite/deploy/kubernetes/scripts/kind-setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
As far as i see this file is a copy `.ci/scripts/kind-setup.sh` but that files has not been deleted yet